### ISSUE TITLE
今天执行了flutter Pub upgrade ，然后运行到ios项目时编译失败 发现是image_gallery_saver包报错

### ISSUE THREAD

**Issue by John-tp** [hui-z/image_gallery_saver#117]
查看pubspec.lock文件后发现image_gallery_saver版本变成了1.6.0，改成image_gallery_saver: 1.5.0后更新插件再运行就OK了

**Comment by houdayec** [comment]
v1.6.0 was released with errors when using Swift 5.1.

Use latest version on master:

```
image_gallery_saver:
    git:
      url: https://github.com/hui-z/image_gallery_saver.git
      ref: master
```

**Comment by xqqlv** [comment]
update v1.6.1
